crates/core/tedge_actors/src/examples/calculator.rs

```diff
@@ -58,130 +57,3 @@ impl Actor for Calculator {
         Ok(())
     }
 }
-
```

Comment:
I didn't really get why you moved all this to a different module. Everything was pretty self contained with the `Server` definition also here. The `Player` probably was an outlier. But this is also fine. Just wondering if I missed something subtle.

Reply:
I moved in two independent crates the two implementations of the same example using respectively the `Actor` and the `Server` trait to avoid the confusion.

- If the same struct implements the `Actor` and the `Service` traits , then the server actor has two messages boxes.
- If the struct that implement the `Actor` trait owns a `Service` implementation (as you proposed), then things are cleaner but confusing.